### PR TITLE
Rewrite bazil.org import paths to suffuse/fuse.

### PR DESCRIPTION
**Pull request by paulp**

### CODE REVIEW

File: fs/fstestutil/record/record.go [hunk 1]
```diff
@@ -1,11 +1,11 @@
-package record // import "bazil.org/fuse/fs/fstestutil/record"
+package record // import "github.com/suffuse/fuse/fs/fstestutil/record"
```
Comment (EECOLOR) [inline fs/fstestutil/record/record.go:1]: That's a bit lazy isn't it. you could have removed the comment.

File: fs/serve.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // FUSE service loop, for servers that wish to use it.
 
-package fs // import "bazil.org/fuse/fs"
+package fs // import "github.com/suffuse/fuse/fs"
```
Comment (EECOLOR) [inline fs/serve.go:3]: Same here

File: fuse.go [hunk 3]
```diff
@@ -98,7 +98,7 @@
 // Behavior and metadata of the mounted file system can be changed by
 // passing MountOption values to Mount.
 //
-package fuse // import "bazil.org/fuse"
+package fuse // import "github.com/suffuse/fuse"
```
Comment (EECOLOR) [inline fuse.go:101]: Ohw

File: fs/fstestutil/record/record.go (hunk 1 above)
Comment (paulp) [inline fs/fstestutil/record/record.go:1]: It is a mistake to mix automated transformations and manual changes.

File: fuseutil/fuseutil.go [hunk 4]
```diff
@@ -1,7 +1,7 @@
-package fuseutil // import "bazil.org/fuse/fuseutil"
+package fuseutil // import "github.com/suffuse/fuse/fuseutil"
```
Comment (EECOLOR) [inline fuseutil/fuseutil.go:1]: It seems that each file has a reminder on how to import that specific package. If you're forking, I recommend you remove it. If someone does do not know how to import things, they will probably not be looking at the source.

File: syscallx/doc.go [hunk 5]
```diff
@@ -10,4 +10,4 @@
 //
 // Options can be implemented with separate wrappers, in the style of
 // Linux getxattr/lgetxattr/fgetxattr.
-package syscallx // import "bazil.org/fuse/syscallx"
+package syscallx // import "github.com/suffuse/fuse/syscallx"
```
Comment (EECOLOR) [inline syscallx/doc.go:13]: Yeah

**Comment by EECOLOR** [comment]
Other than the above it looks fine and you're good to go

File: fs/fstestutil/record/record.go (hunk 1 above)
Comment (EECOLOR) [inline fs/fstestutil/record/record.go:1]: Is that something I don't understand, something you did or something bazil did?

Comment (paulp) [inline fs/fstestutil/record/record.go:1]: It's something you say I should have done. Removing the comment is a manual change. The  commit has no manual changes.

Comment (EECOLOR) [inline fs/fstestutil/record/record.go:1]: Ahh I see what you mean now. Then you should have probably left out manual the changes to `.travis.yml` and `README.md`.

Anyway. No need to argue about it. I understand the 'keep automated transformations in their own commit' philosophy'

Comment (paulp) [inline fs/fstestutil/record/record.go:1]: The change to .travis.yml accompanies it because it is affected by it. The README I could have left but I don't overthink the commit history of documentation.

**Comment by EECOLOR** [comment]
I forgot to press the button
